packages/harpguru-core/src/components/harp-face/hooks/use-styles.ts

```diff
@@ -1,28 +1,30 @@
 import { StyleSheet } from 'react-native'
 import type { ViewStyle } from 'react-native'
-import type { HarpStrata } from 'harpstrata'
 
-import { getHarpFaceFacts } from '../../../utils'
-import { useSizes } from '../../../styles'
+import { useLayoutFacts, useSizes } from '../../../styles'
+
+import { useOctaveColumnGroups } from './use-octave-column-groups'
 
 type HarpFaceStyles = {
   readonly face: ViewStyle
 }
 
-export const useStyles = (activeHarpStrata: HarpStrata): HarpFaceStyles => {
+export const useStyles = (): HarpFaceStyles => {
   const { dynamicSizes } = useSizes()
   const { columnWidth, rowHeight, fragmentGutter } = dynamicSizes
-  const { columnCount, rowCount, octaveColumnGroups } = getHarpFaceFacts(
-    activeHarpStrata
-  )
+
+  const { harpFaceRowCount, harpFaceColumnCount } = useLayoutFacts()
+
+  const octaveColumnGroups = useOctaveColumnGroups()
   const { length: groupCount } = octaveColumnGroups
 
   const styles = StyleSheet.create({
     face: {
       flexDirection: 'row',
       justifyContent: 'space-around',
-      width: columnWidth * columnCount + (fragmentGutter * groupCount + 1),
-      height: rowHeight * rowCount,
+      width:
```

Comment:
Why is this `+1`? Is that a hacky workaround to some scaling issue?

Reply:
Raised as techdebt in #113 

The work done to make the app sizing dynamic (at #107) was used as a platform for additional speculative pull requests to ensure that this was the correct direction to start down.

Rather than addressing the finer details in this or subsequent pull request and potentially struggling to merge up to those subsequent branches I will prefer to record the issues in each merge request and resolve them afterwards.

This has a limit of course. Major issues will not be merged to master. I will need to find a way around that if it happens.